### PR TITLE
update peerDependencies angular to support version 15.2

### PR DESCRIPTION
**Pull request by davidgvf**

### CODE REVIEW

File: projects/ngx-cookiebot/package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "url": "https://github.com/halloverden/ngx-cookiebot/tree/main/projects/ngx-cookiebot"
   },
   "peerDependencies": {
-    "@angular/common": "8.2.14 - 14",
-    "@angular/core": "8.2.14 - 14"
+    "@angular/common": "8.2.14 - 15.2",
```
Comment (andreaslarssen) [inline projects/ngx-cookiebot/package.json:33]: Could you target the major version?

File: projects/ngx-cookiebot/package.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
...
+    "@angular/core": "8.2.14 - 15.2"
```
Comment (andreaslarssen) [inline projects/ngx-cookiebot/package.json:34]: Could you target the major version?

**Review by andreaslarssen — CHANGES_REQUESTED** [review]

File: projects/ngx-cookiebot/package.json (hunk 1 above)
Comment (davidgvf) [reply projects/ngx-cookiebot/package.json]: yes

**Comment by andreaslarssen** [comment]
@davidgvf nice, did you test it?

**Comment by andreaslarssen** [comment]
@davidgvf ok, will provide a release candidate for you to test as soon as I can

**Comment by davidestrim** [comment]
@andreaslarssen when can you do it?

**Comment by andreaslarssen** [comment]
> @andreaslarssen when can you do it?

This evening I was hoping

**Comment by andreaslarssen** [comment]
@davidgvf The 2.3.0-rc.1 version is out. Try that and let me know if it looks good. I will also test it on my side, and I'll publish 2.3.0 after we've both verified

**Comment by andreaslarssen** [comment]
@davidestrim / @davidgvf 2.3.0 is out

**Pull request merged**